pymc/distributions/censored.py

```diff
@@ -49,6 +49,16 @@ class Censored(SymbolicDistribution):
     upper: float or None
         Upper (right) censoring point. If `None`, the distribution will not be right censored.
 
+    Notes
```

Comment:
I think this should be a [warnings](https://numpydoc.readthedocs.io/en/latest/format.html#warnings) section with raw text and no admonition. 

It will look like: https://arviz-devs.github.io/arviz/api/generated/arviz.reloo.html (that also has a notes section)